makefile

```diff
@@ -190,7 +190,7 @@ include $(wildcard $(patsubst %.o,%.d,$(demoLibControls_OBJS)))
 
 ## OPHD project ##
 
-ophd_SRCDIR := OPHD/
+ophd_SRCDIR := appOPHD/
```

Comment:
Wondering if this should be squashed into the folder rename commit. Normally I try to keep each commit buildable on it's own.